security/pkg/stsservice/tokenmanager/google/tokenexchangeplugin.go

```diff
@@ -306,6 +310,38 @@ func (p *Plugin) fetchFederatedToken(parameters security.StsRequestParameters) (
 	return respData, nil
 }
 
+// ExchangeToken exchanges oauth access token from K8s SA jwt.
+func (p *Plugin) ExchangeToken(k8sSAjwt string) (string, error) {
```

Comment:
Is this function replacing "func (p *SecureTokenServiceExchanger) ExchangeToken(k8sSAjwt string)"? In original ExchangeToken(), there is a retry mechanism. Why is the retry mechanism removed?

func (p *SecureTokenServiceExchanger) ExchangeToken(k8sSAjwt string) (string, error) {
         ....
	body, err := p.requestWithRetry(jsonStr)
         ...
}